package.json

```diff
@@ -59,7 +57,7 @@
     "multer": "^1.4.2",
     "node-foursquare": "^0.4.6",
     "node-linkedin": "^0.5.6",
-    "node-sass": "^4.13.1",
+    "node-sass": "^4.14.1",
```

Comment:
In many other projects we've switched from `node-sass` to `sass`, but if this is working there's no reason to change.

Reply:
Right, but mostly in front-end apps where it's used by the webpack loader. Here it's used by node-sass-middleware / express.js server. So, I wasn't sure if it works with the other implementation.